src/profession.cpp

```diff
@@ -462,6 +469,12 @@ std::vector<bionic_id> profession::CBMs() const
     return _starting_CBMs;
 }
 
+std::vector<proficiency_id> profession::
+proficiencies() const
```

Comment:
Putting a line break here is not in keeping with the general style of the codebase.

Reply:
I didn't intend to put that there, I'm not sure why it is.